packages/paste-website/src/pages/getting-started/design/index.mdx

```diff
@@ -0,0 +1,529 @@
+---
+title: Design Guidelines for Paste
+description: Notes on how to use Paste in design work
+slug: /getting-started/design/
+---
+
+import {graphql} from 'gatsby';
+import {Anchor} from '@twilio-paste/anchor';
+import {Box} from '@twilio-paste/box';
+import {Text} from '@twilio-paste/text';
+import {Paragraph} from '@twilio-paste/paragraph';
+import {Card} from '@twilio-paste/card';
+import {Disclosure, DisclosureHeading, DisclosureContent} from '@twilio-paste/disclosure';
+import {Tooltip} from '@twilio-paste/tooltip';
+import {Flex} from '@twilio-paste/flex';
+import {Stack} from '@twilio-paste/stack';
+import {version as typesVersion} from '@twilio-paste/types/package.json';
```

Comment:
I assume there is a lot of copy and paste from another page. Can you make sure that only the imports you are using remain, deleting everything else that isn't needed?

Reply:
lol oops, this is fixed now